README.md

```diff
@@ -294,6 +294,38 @@ curl -X PUT -H "Authorization: ${TOKEN}" "http://localhost/irods-rest/0.9.1/put_
 **Returns**
 Nothing on success
 
+### /meta
+This endpoint simulates a subset of the "imeta" family of icommands: add, rm, and set.
+**Method**: GET
+**Parameters**
+- target: An absolute logical path or entity name specifying the recipient of the metadata.
+- entity_type: Flag specifiying Data Object ("d"), Collection("C"), Resource("R"), or User ("u"), following the imeta command's conventions.
+- attrib: Metadata attribute
+- value: Metadata value
+- unit: Metadata unit
```

Comment:
Will investigate -- I think we get things out of Pistache requests as boost::optionals

Reply:
In iRODS, the units are optional. This endpoint must not force users to pass units.

Reply:
Leaving this open only in case my solution (which was to pass an empty string as the unit in case the user doesn't pass one) is not in fact appropriate.